src/scene_graph/camera_node.cpp

```diff
@@ -10,12 +10,26 @@ namespace wr
 		SignalChange();
 	}
 
+	void CameraNode::SetFov(float aspect_ratio, float filmSize)
+	{
+		float verticalSize = filmSize / aspect_ratio;
+		m_fov = 2.0f * std::atan2(verticalSize, 2.0f * m_focal_length);
+		SignalChange();
```

Comment:
This calculation changed I believe. Can you explain why this exactly? We might want to keep the old FoV approach available since that is more common in games for example.

Reply:
This is an overloaded function, but to avoid confusion I will change the function name to "SetFovFromFocalLength"